sgp30/sgp30.h

```diff
@@ -33,6 +33,7 @@
 #include "sensirion_arch_config.h"
 #include "sensirion_common.h"
 #include "sensirion_i2c.h"
+#include "git_version.h"
```

Comment:
Is there a reason to include it here instead of in the sgp30.c file where it is used? I don't see any problem in it, just curious if there is some kind of related style guide rule or anything.

Reply:
I agree, the include should be in the c file, I don't see a reason why they should be in the h file

Reply:
sgp30.h is the public interface, as such, the idea was to make the `#define SGP_DRV_VERSION_STR` available publicly, but there's already the getter function that's exposed and it's probably better not to expose the define directly, so I agree we can indeed remove it.